addons/stock/models/stock_quant.py

```diff
@@ -964,7 +964,7 @@ class QuantPackage(models.Model):
     quant_ids = fields.One2many('stock.quant', 'package_id', 'Bulk Content', readonly=True,
         domain=['|', ('quantity', '!=', 0), ('reserved_quantity', '!=', 0)])
     package_type_id = fields.Many2one(
-        'stock.package.type', 'Package Type', index=True, check_company=True)
+        'stock.package.type', 'Package Type', index=True)
```

Comment:
I think we want to avoid that. This model doesn't have _check_company_auto = True so we should maybe just modify the call to _check_company to a better place in order to correct this issue but prevent behavior like tic describe

Reply:
Not sure it still use in fact. Gonna clean that in another task